frontend/integration-tests-cypress/cypress-e2e/cypress/plugins/index.js

```diff
@@ -0,0 +1,38 @@
+// ***********************************************************
```

Comment:
Shall we remove the "This example plugin...." comment as you've customized it?

Reply:
fixed